docs/learn/advanced/privacy.md

```diff
@@ -0,0 +1,20 @@
+# Privacy
+This page describes various privacy aspects of the Ceramic protocol. Measures we've taken so far, planned future measures, and future research directions.
+
+## Ceramic Protocol
+The StreamType system is already quite flexible in terms of how streams can provide developers with privacy features.
+
+### Default properties of Tile and Caip10Links
+Currently if you create a TileDocument or Caip10Link stream any data put into it will be public by default. It's however possible to encrypt the content put into the stream by accessing the DID instance from Ceramic (`ceramic.did`) and using it's encryption functionality as described in the [How to store signed and encrypted data on IPFS](https://blog.ceramic.network/how-to-store-signed-and-encrypted-data-on-ipfs/) blog post.
```

Comment:
```suggestion
Currently if you create a TileDocument or Caip10Link stream any data put into it will be public by default. It's however possible to encrypt the content put into the stream by accessing the DID instance from Ceramic (`ceramic.did`) and using its encryption functionality as described in the [How to store signed and encrypted data on IPFS](https://blog.ceramic.network/how-to-store-signed-and-encrypted-data-on-ipfs/) blog post.
```